### PR TITLE
Fixes #2, Windows support

### LINKED ISSUE #2: Add basic support for phony targets
Current implementation crashes on the following Makefile

```
.PHONY: all

all: file

file:
    touch file
```

if  `all` exists and `file` does not. Error is the following: `stat /tmp/file: no such file or directory`

We should add at least basic support for `.PHONY` targets.

Initially this issue was discovered while tried to graph https://github.com/netty/netty that contains `all` folder.

Action plan:
- Add support for .PHONY to `makex`
- Mark `all` as `.PHONY` in  `srclib`
- Update `srclib` and `sourcegraph` dependencies (`makex`, `srclib`)

### PR DESCRIPTION
**Pull request by alexsaveliev**
- fixes #2 by adding basic .PHONY target support
- Make it compile and pass tests on Windows

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 install: ${GOBIN}/makex
 
-${GOBIN}/makex: $(shell find -type f -and -name '*.go')
+${GOBIN}/makex: $(shell /usr/bin/find -type f -and -name '*.go')
```
Comment (beyang) [inline Makefile]: Is this necessary for Windows? On some *nix systems, this might be located at `/usr/local/bin/find` or elsewhere.

**Comment by beyang** [comment]
aside from comment, LGTM.

File: Makefile (hunk 1 above)
Comment (alexsaveliev) [inline Makefile]: Yes, on Windows in 99% of cases `\Windows\System32\find.exe` will be used (performs text search). I doubt that somebody wants to use `find` from `/usr/local` or other non-standard script/binary from the `PATH` and I think that `/usr/bin` is the correct location everywhere

Comment (beyang) [inline Makefile]: Add a check for OS and use `find` for non-Windows and the full path for Windows.

Comment (alexsaveliev) [inline Makefile]: > Add a check for OS and use find for non-Windows and the full path for Windows.

Done

**Pull request merged**